cmd/main.go

```diff
@@ -95,12 +96,23 @@ func main() {
 		os.Exit(1)
 	}
 
+	vspImages := make(map[string]string)
+
+	for _, vspImageName := range plugin.VspImages {
+		value := os.Getenv(vspImageName)
```

Comment:
we're also doing `os.Getenv(vspImageName)` elsewhere. Need to change this in the future so that we only do it in 1 place.